src/nfa/nfa.cc

```diff
@@ -322,6 +322,37 @@ void Delta::add(State state_from, Symbol symbol, State state_to) {
     }
 }
 
+void Delta::add_set(State state_from, Symbol symbol, const StateSet& states) {
+    if(states.size() == 0) {
```

Comment:
Use a more descriptive function.
```suggestion
    if(states.empty()) {
```